atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -63,31 +66,61 @@ public SchemaMutationLockTables(CassandraClientPool clientPool, CassandraKeyValu
     }
 
     public TableReference createLockTable() throws TException {
-        return clientPool.runWithRetry(this::createLockTable);
-    }
-
-    private TableReference createLockTable(CassandraClient client) throws TException {
-        String lockTableName = LOCK_TABLE_PREFIX + "_" + getUniqueSuffix();
-        TableReference lockTable = TableReference.createWithEmptyNamespace(lockTableName);
+        TableReference lockTable = TableReference.createWithEmptyNamespace(LOCK_TABLE_PREFIX);
         log.info("Creating lock table {}", SafeArg.of("schemaMutationTableName", lockTable));
-        createTableInternal(client, lockTable);
+        createTableWithCustomId(lockTable);
         return lockTable;
     }
 
-    private String getUniqueSuffix() {
-        // We replace '-' with '_' as hyphens are forbidden in Cassandra table names.
-        return UUID.randomUUID().toString().replace('-', '_');
-    }
+    private void createTableWithCustomId(TableReference tableRef) throws TException {
+        String internalTableName = CassandraKeyValueServiceImpl.internalTableName(tableRef);
+        String keyspace = config.getKeyspaceOrThrow();
+        String fullTableNameForUuid = keyspace + "." + internalTableName;
+        UUID uuid = UUID.nameUUIDFromBytes(fullTableNameForUuid.getBytes());
+
+        // TODO use parameters as in ColumnFamilyDefinitions.getStandardCfDef
+        String createTableStatement = "CREATE TABLE \"%s\".\"%s\" (\n"
+                + "    key blob,\n"
+                + "    column1 blob,\n"
+                + "    column2 bigint,\n"
+                + "    value blob,\n"
+                + "    PRIMARY KEY (key, column1, column2)\n"
+                + ") WITH COMPACT STORAGE\n"
+                + "    AND CLUSTERING ORDER BY (column1 ASC, column2 ASC)\n"
+                + "    AND bloom_filter_fp_chance = 0.1\n"
+                + "    AND caching = '{\"keys\":\"ALL\", \"rows_per_partition\":\"NONE\"}'\n"
+                + "    AND comment = ''\n"
+                + "    AND compaction = {'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'}\n"
+                + "    AND compression = {}\n"
+                + "    AND dclocal_read_repair_chance = 0.1\n"
+                + "    AND default_time_to_live = 0\n"
+                + "    AND gc_grace_seconds = 3600\n"
+                + "    AND max_index_interval = 2048\n"
+                + "    AND memtable_flush_period_in_ms = 0\n"
+                + "    AND min_index_interval = 128\n"
+                + "    AND read_repair_chance = 0.0\n"
+                + "    AND speculative_retry = 'NONE'"
+                + "    AND id = '%s'";
+        CqlQuery query = new CqlQuery(createTableStatement,
+                SafeArg.of("keyspace", keyspace),
+                SafeArg.of("internalTableName", internalTableName),
```

Comment:
If this is general we should use LoggingArgs, though at this point I'm not sure if the schema metadata has been loaded yet. Of course if we follow @tpetracca's suggestion to be specific to the schema lock table (and I think we should), then saying that is safe is of course fine.